src/exports/react.ts

```diff
@@ -10,7 +10,7 @@ export function useAccount(wallet: Pick<AztecWalletSdk, "accountObservable">) {
       setAccount(account);
     });
     return () => unsubscribe();
-  }, []);
+  }, [wallet.accountObservable]);
```

Comment:
The subscription would get stale upon sdk chamge without this.

Reply:
good catch. Did you test that it works?